### PR TITLE
Add needs_triage label to new issues/PRs by default

### PR DESCRIPTION
**Pull request by NilashishC**
##### SUMMARY
- Allow issues/PRs to be labelled `needs_triage` by default when created.
- This would facilitate the triager tool to be more efficient.
- This also overrides the ansible-collection org level .github/, hence the entire directory has been copied over and only `labels: needs_triage` has been added.

### CODE REVIEW

**Comment by NilashishC** [comment]
recheck

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# https://probot.github.io/apps/settings/
+#
+# Used by some over repos via https://github.com/ansible-collections/community.general/blob/master/.github/settings.yml
+
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+  has_issues: true
+  has_wiki: false
+  has_pages: false
+  default_branch: devel
```
Comment (Qalthos) [inline .github/settings.yml:11]: I'm pretty sure you mean
```suggestion
  default_branch: main
```

**Comment by ganeshrn** [comment]
recheck

**Comment by trishnaguha** [comment]
@NilashishC Are we getting this in?

**Comment by trishnaguha** [comment]
@NilashishC Can we add P1, P2 labels as well?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+##### ISSUE TYPE
+- Bug Report
+
+##### COMPONENT NAME
+<!--- Write the short name of the module, plugin, task or feature below, use your best guess if unsure -->
+
+##### ANSIBLE VERSION
+<!--- Paste verbatim output from "ansible --version" between quotes -->
+```paste below
+
```
Comment (Qalthos) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Collection version?

**Pull request closed without merging**